runtime/controller_test.go

```diff
@@ -1390,7 +1390,8 @@ name: my-gcs
 		},
 		Resource: &runtimev1.Resource_Connector{
 			Connector: &runtimev1.ConnectorV2{
-				Spec: &runtimev1.ConnectorSpec{Driver: "s3", Properties: map[string]string{"region": "us-west-2"}},
+				Spec:  &runtimev1.ConnectorSpec{Driver: "s3", Properties: map[string]string{"region": "us-west-2"}},
+				State: &runtimev1.ConnectorState{SpecHash: "76b3819c673b0a4e4755418614d8cb62"},
```

Comment:
You can `nil` the field here to omit it from tests: https://github.com/rilldata/rill/blob/abb730b9e8843b7f912e7010c4edd3e42ace22f6/runtime/testruntime/reconcile.go#L155